server/src/helpers/errorHandler.helper/errorHandler.helper.ts

```diff
@@ -0,0 +1,24 @@
+import express from 'express';
+import HttpStatusCode from './httpStattusCode.helper';
+
+export class ErrorHandler extends Error {
```

Comment:
Can "ErrorHandler" class be renamed to something like "CustomError"?

Reply:
I think it's better to leave it as it is. The developer will start typing Error and the IDE will help him.  No need to remember what it was named to start typing

Reply:
@taras-dubyk May be ErorResponse?

Reply:
Let it be, leave ErrorHandler for now